Tests/CMakeLists.txt

```diff
@@ -110,6 +110,14 @@ function(add_test_rev TEST_NAME TEST_EXE_DIR)
     set_tests_properties(${TEST_NAME} PROPERTIES LABELS "regression;verification;no-ci")
 endfunction(add_test_rev)
 
+# Regression test with case-specific verification test.py
+function(add_test_rt TEST_NAME TEST_EXE_DIR)
+    setup_test()
+    set(RUNTIME_OPTIONS "amr.max_step=10 peleLM.do_temporals=1 peleLM.do_extremas=1 peleLM.temporal_int=1 ${RUNTIME_OPTIONS}")
+    add_test(${TEST_NAME} sh -c "${CURRENT_TEST_EXE} ${MPIEXEC_POSTFLAGS} ${CURRENT_TEST_BINARY_DIR}/${TEST_NAME}.inp ${RUNTIME_OPTIONS} > ${TEST_NAME}.log ${SAVE_GOLDS_COMMAND} ${FCOMPARE_COMMAND} && nosetests ${CURRENT_TEST_BINARY_DIR}/test.py")
```

Comment:
In order to pass the ubuntu CI test, I had to remove `${MPI_COMMANDS}` from `add_test()`.  This is a work around for now as I don't think the problem is related to this PR, but more so related to the MPI commands in the ubuntu build.